### PR TITLE
Invalid messages description

### PR DESCRIPTION
**Pull request by tisnik**
# Description

Invalid messages description

## Type of change

- Documentation update

## Testing steps

N/A

### CODE REVIEW

**Review by Bee-lee — APPROVED** [review]
this is great

File: README.md [hunk 1]
```diff
@@ -105,6 +105,72 @@ Please look into [this document](messages/broken/README.md) for detailed descrip
 
 
 
+#### Messages with invalid format - missing account number
+
+These messages should be rejected by *Insights Results Writer* and also by *CCS
```
Comment (joselsegura) [inline README.md:110]: CCS -> CCX if I'm not wrong

File: README.md [hunk 2]
```diff
@@ -105,6 +105,72 @@ Please look into [this document](messages/broken/README.md) for detailed descrip
...
+
+These messages are stored in `messages/invalid_no_account_number/` subdirectory.
+
+Please look into [this document](messages/invalid_no_account_number/README.md) for detailed description.
+
+
+
+#### Messages with invalid format - missing cluster name
+
+These messages should be rejected by *Insights Results Writer* and also by *CCS
```
Comment (joselsegura) [inline README.md:121]: CCS -> CCX if I'm not wrong

File: README.md [hunk 3]
```diff
@@ -105,6 +105,72 @@ Please look into [this document](messages/broken/README.md) for detailed descrip
...
+
+These messages are stored in `messages/invalid_no_cluster_name/` subdirectory.
+
+Please look into [this document](messages/invalid_no_cluster_name/README.md) for detailed description.
+
+
+
+#### Messages with invalid format - missing last checked timestamp
+
+These messages should be rejected by *Insights Results Writer* and also by *CCS
```
Comment (joselsegura) [inline README.md:132]: CCS -> CCX if I'm not wrong

File: README.md [hunk 4]
```diff
@@ -105,6 +105,72 @@ Please look into [this document](messages/broken/README.md) for detailed descrip
...
+
+These messages are stored in `messages/invalid_no_last_checked/` subdirectory.
+
+Please look into [this document](messages/invalid_no_last_checked/README.md) for detailed description.
+
+
+
+#### Messages with invalid format - missing organization ID
+
+These messages should be rejected by *Insights Results Writer* and also by *CCS
```
Comment (joselsegura) [inline README.md:143]: CCS -> CCX if I'm not wrong

File: README.md [hunk 5]
```diff
@@ -105,6 +105,72 @@ Please look into [this document](messages/broken/README.md) for detailed descrip
...
+
+These messages are stored in `messages/invalid_no_organization_id/` subdirectory.
+
+Please look into [this document](messages/invalid_no_organization_id/README.md) for detailed description.
+
+
+
+#### Messages with invalid format - missing version info
+
+These messages should be rejected by *Insights Results Writer* and also by *CCS
```
Comment (joselsegura) [inline README.md:154]: CCS -> CCX if I'm not wrong

File: README.md [hunk 6]
```diff
@@ -105,6 +105,72 @@ Please look into [this document](messages/broken/README.md) for detailed descrip
...
+
+These messages are stored in `messages/invalid_no_version/` subdirectory.
+
+Please look into [this document](messages/invalid_no_version/README.md) for detailed description.
+
+
+
+#### Messages with invalid format - missing report
+
+These messages should be rejected by *Insights Results Writer* and also by *CCS
```
Comment (joselsegura) [inline README.md:165]: CCS -> CCX if I'm not wrong

**Review by joselsegura — CHANGES_REQUESTED** [review]
Probably a mistake when copy&pasting :-D

**Review by joselsegura — APPROVED** [review]

**Pull request merged**